OpenSim/Simulation/Model/PathActuator.cpp

```diff
@@ -206,22 +205,6 @@ double PathActuator::computeMomentArm(const SimTK::State& s, Coordinate& aCoord)
 }
 
 //------------------------------------------------------------------------------
-//                            CONNECT TO MODEL
-//------------------------------------------------------------------------------
-/**
- * Perform some setup functions that happen after the
- * object has been deserialized or copied.
- *
- * @param aModel OpenSim model containing this PathActuator.
- */
-void PathActuator::extendFinalizeFromProperties()
-{
-    GeometryPath &path = updGeometryPath();
```

Comment:
`path` was not used, but since `extendFinalizeFromProperties()` doesn't do anything, I removed it altogether.